themes/bootstrap3/templates/myresearch/historicloans.phtml

```diff
@@ -0,0 +1,131 @@
+<?
+  // Set up page title:
+  $this->headTitle($this->translate('Loan History'));
+
+  // Set up breadcrumbs:
+  $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li> <li class="active">' . $this->transEsc('Loan History') . '</li>';
+?>
+
+<div class="<?=$this->layoutClass('mainbody')?>">
+  <h2><?=$this->transEsc('Your Loan History')?></h2>
+  <?=$this->flashmessages()?>
+
+  <?=$this->context($this)->renderInContext('librarycards/selectcard.phtml', ['user' => $this->auth()->isLoggedIn()]); ?>
+
+  <? if (!empty($this->transactions)): ?>
+    <nav class="search-header hidden-print">
+      <? if ($this->paginator): ?>
+        <div class="search-stats">
+          <?
+            $end = min(
+              $this->paginator->getAbsoluteItemNumber($this->paginator->getItemCountPerPage()),
+              $this->paginator->getTotalItemCount()
+            );
+            $transParams = [
+              '%%start%%' => $this->localizedNumber($this->paginator->getAbsoluteItemNumber(1)),
+              '%%end%%'   => $this->localizedNumber($end),
+              '%%total%%' => $this->localizedNumber($this->paginator->getTotalItemCount())
+            ];
+          ?>
+          <?=$this->translate('showing_items_of_html', $transParams); ?>
+        </div>
+      <? endif; ?>
+      <? if ($this->sortList): ?>
+        <?=$this->context($this)->renderInContext('Helpers/sort.phtml', ['sortList' => $this->sortList]); ?>
```

Comment:
I see you've created a new helper template here... can this also be used elsewhere, like the checked out item list? Is "Helpers" the right place to put this (vs., for example, myresearch/controls/sort.phtml, which would be more consistent with how we handle this on the search side)?

Reply:
I put it there because pagination.phtml is also there and both are generic enough to be used anywhere. sort.phtml is actually a partial copy of search/controls/sort.phtml, so it might make sense to make search/controls/sort.phtml include the generic sort.phtml and add myresearch/controls/sort.phtml that does the same. Would that make sense to you?

Reply:
Yes, that makes sense to me, assuming that refactoring search/controls/sort.phtml is workable. Worth a try, anyway! In the end, what will the difference be between search/controls/sort and myresearch/controls/sort? Will both render the same control but use different mechanisms to extract the necessary values?

Reply:
I tried it but it didn't work out to be as simple as I would have liked. In search the values come from the Params object and then there are the hidden fields for the current search terms etc. In the end I decided to do what you suggested and moved Helpers/sort.phtml to myresearch/controls/sort.phtml and kept them separate.

Reply:
Thanks! I'll give this another test/review in the near future.